src/components/mol.detail-user/detail-user.component.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { UseDetailsUsers } from "../../domain/user-details-use-case";
```

Comment:
por convenção os hooks tem o prefixo `use` minúsculo
```suggestion
import { useDetailsUsers } from "../../domain/user-details-use-case";
```

Reply:
ajustado